admin/build_targets/ubuntu-15.10/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright Hybrid Logic Ltd. See LICENSE file for details.
```

Comment:
PLEASE NOTE: 
clusterhqci/fpm-ubuntu-wily is an image I have created manually. It looks ok, but I haven't checked it in anywhere.
This manually-built docker image is temporary, as we need to update Jenkins so it builds it periodically the same way it does with Ubuntu14.04 and CentOS (and Ubuntu15.04, but we need to delete that bit as it is not useful any more)